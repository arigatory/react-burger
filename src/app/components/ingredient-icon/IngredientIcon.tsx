import React from 'react';
import styles from './ingredientIcon.module.scss';

interface Props {
  img: string;
  text?: string;
}

export default function IngredientIcon({ img, text }: Props) {
  return (
      <div className={styles.img}>
        <img src={img} alt={img} />
      </div>
  );
}
