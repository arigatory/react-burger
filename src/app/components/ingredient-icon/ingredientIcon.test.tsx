import renderer from 'react-test-renderer';

import IngredientIcon from './IngredientIcon';

it('Рисует иконку ингредиента бургера', () => {
  const tree = renderer
    .create(
      <IngredientIcon img="https://code.s3.yandex.net/react/code/bun-02.png" />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
