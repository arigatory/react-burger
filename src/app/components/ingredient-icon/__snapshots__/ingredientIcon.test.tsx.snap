// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Рисует иконку ингредиента бургера 1`] = `
<div
  className="img"
>
  <img
    alt="Ingredient icon"
    src="https://code.s3.yandex.net/react/code/bun-02.png"
  />
</div>
`;
